### ISSUE TITLE
Expected training time on cityscape dataset

### ISSUE THREAD

**Issue by ChristianEschen** [itijyou/ademxapp#11]
I am trying to train the cityscape model

However, it looks like my training gets stucked. I am using 1X k40C 12 GB and 1X Titan X 12 GB. with batch size equal 200 and crop size 100.
The last print from terminal was:
 2017-02-10 00:27:06,021 Host Epoch[0] Batch [4]	Speed: 24.24 samples/sec	Train-fcn_valid=0.270395

Does anyone know the expected training time on this data?

**Comment by itijyou** [comment]
The default configuration may need several (probably 3) days.

**Comment by ChristianEschen** [comment]
Thank you.
I don't know if the network is still training. It is not still not printing training validation accuracy. 
The last print was 2017-02-10 00:27:06,021 Host Epoch[0] Batch [4] Speed: 24.24 samples/sec Train-fcn_valid=0.270395.
Have you observed issues like this before?
I have enough free memory.

**Comment by itijyou** [comment]
When there are other running processes using GPUs at the same time, there would be a chance to get stuck. When reading images too fast from a relatively slow filesystem (e.g., HDD), there'd be a chance too, which seems like your case.
But I did not manage to find the real cause of this problem. :smiley:
Any clues or suggestions to solve this problem are welcome.

**Comment by ChristianEschen** [comment]
For some reason, all the training files were not transfered to the database path. 
Downloading the data again solved the problem.

**Comment by ChidanandKumarKS** [comment]
hai all, what is the command to execute training procedure for cityscape semantic segmentation? any updates on training codes for semantic segmentation?

**Comment by ChidanandKumarKS** [comment]
Hai Christian, Kindly suggest  command to execute training procedure for cityscape semantic segmentation.
